docs/sphinx/source/clearsky.rst

```diff
@@ -118,6 +114,57 @@ The Ineichen and Perez clear sky model parameterizes irradiance in terms
 of the Linke turbidity [Ine02]_. pvlib-python implements this model in
 the :py:func:`pvlib.clearsky.ineichen` function.
 
+
+The :py:func:`~pvlib.atmosphere.kasten96_lt` function can be used to calculate
+Linke turbidity [Kas96]_ as input to the clear sky Ineichen and Perez function.
+The Kasten formulation requires precipitable water and broadband AOD which can
```

Comment:
perhaps spell out AOD at first use

Reply:
Two sentences here, please. Perhaps split after AOD.

Reply:
changed to:
>The Kasten formulation requires precipitable water and broadband aerosol
optical depth (AOD). According to Molineaux, broadband AOD can be approximated
by a single measurement at 700-nm `[Mol98]_`. An alternate broadband AOD
approximation from Bird and Hulstrom combines AOD measured at two
wavelengths `[Bir80]_`, and is implemented in
`:py:func:``~pvlib.atmosphere.bird_hulstrom80_aod_bb`` `.

better?

Reply:
yes, thanks